src/Arr.php

```diff
@@ -726,4 +726,63 @@ public function isEmpty(): bool
     {
         return $this->length() === 0;
     }
+
+    /**
+     * Returns a string representing the elements of the array
+     * @return string the string representation
+     */
+    public function toLocaleString(string $locale = 'en_US', string $timezone = 'UTC'): string
```

Comment:
By the way, while this covers the `Date.prototype.toLocaleString()` there is also the `Object.prototype.toLocaleString()` and the `Number.prototype.toLocalString()` that can be converted from the `Array.protoype.toLocalString()`.

Link to the other types of `toLocalString()` conversions
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toLocaleString#examples

Reply:
I can definitely add support for the other methods, I'll update the current PR unless otherwise advised.

Reply:
I agree. if you want you can do it in this PR, or if you prefer you could close this and create another new PR with this refactor.
I mention "refactor" because this package is for Array so I would like to provide the toLocaleString for Arr (public method). The toLocaleString for other types (number and data) is for a clear code that is easier to test.